src/users/UsersProfileScreen.js

```diff
@@ -0,0 +1,19 @@
+/* @flow strict-local */
+import React from 'react';
+import type { Node } from 'react';
+import { SafeAreaView } from 'react-native-safe-area-context';
+
+import type { AppNavigationProp } from '../nav/AppNavigator';
+import UsersProfileCard from './UsersProfileCard';
+
+type Props = $ReadOnly<{|
+  navigation: AppNavigationProp<'users'>,
```

Comment:
This should follow the model of the other screen components that have the same navigation-related context -- which means the ones that also appear in `MainTabsScreen` in the same way that you're adding this one there. Take a look at the `navigation` props on those.